doc/releases/index.rst

```diff
@@ -115,7 +114,20 @@ to be able to understand the context of the change.
    :glob:
    :reversed:
 
-   migration-guide-*
+   migration-guide-3.[6-7]
+   migration-guide-4.[0]
+
+End-of-life Releases
```

Comment:
```suggestion
End-of-life releases
```

Reply:
The document already has consistent capitalization in all the headings so I would suggest to leave as-is